x/bounty/keeper/keeper_test.go

```diff
@@ -0,0 +1,169 @@
+package keeper_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/tendermint/tendermint/crypto/ed25519"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	sdksimapp "github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types1 "github.com/cosmos/cosmos-sdk/types"
+
+	shentuapp "github.com/shentufoundation/shentu/v2/app"
+	"github.com/shentufoundation/shentu/v2/x/bounty/keeper"
+	"github.com/shentufoundation/shentu/v2/x/bounty/types"
+)
+
+var (
+	acc1 = sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address().Bytes())
+	acc2 = sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address().Bytes())
+	acc3 = sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address().Bytes())
+	acc4 = sdk.AccAddress(ed25519.GenPrivKey().PubKey().Address().Bytes())
+)
+
+// shared setup
+type KeeperTestSuite struct {
+	suite.Suite
+	app     *shentuapp.ShentuApp
+	ctx     sdk.Context
+	keeper  keeper.Keeper
+	address []sdk.AccAddress
+	// queryClient types.QueryClient
+}
+
+func (suite *KeeperTestSuite) SetupTest() {
+	suite.app = shentuapp.Setup(false)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{})
+	suite.keeper = suite.app.BountyKeeper
+
+	for _, acc := range []sdk.AccAddress{acc1, acc2, acc3, acc4} {
```

Comment:
Use this method to init address and balance maybe more simple
`suite.address = shentuapp.AddTestAddrs(app, ctx, 4, sdk.NewInt(1e10))`